benchmarks/templates/array.md

```diff
@@ -0,0 +1,22 @@
+# C
+|               | array.Get(), ns | array.Set(), ns |
+| ------------- | ------------: | --------: |
+| Native call | - | - |
+| Proxy-side call | ${array.proxy_array_get.joint(lang:c)[main]} | ${array.proxy_array_set.joint(lang:c)[main]} |
+
+# C++
```

Comment:
Put C++ benchmarks before C. This is not cool, but we have to keep the order consistent between different templates.

Reply:
Okay